spyder/widgets/fileswitcher.py

```diff
@@ -658,17 +658,20 @@ def setup_file_list(self, filter_text, current_path):
         for index, score in enumerate(scores):
             text, rich_text, score_value = score
             if score_value != -1:
-                text_item = "<big style='color:{0:}'>{1:}</big>".format(
-                        ima.MAIN_FG_COLOR, rich_text.replace('&', ''))
+                text_item = ("<span style='font-size: 13pt; color:{0:}'>{1:}"
```

Comment:
Yep you are right @ccordoba12 that's better. Please @juanis2112 instead of using the hard-coded value create variables that get the current rich font size (for the paths) and calculate the size for the filenames, something like:
```python
        path_text_font_size = CONF.get('main', 'rich_font/size', 11)
        filename_text_font_size = path_text_font_size + 2
```
And use these variables in the `formats` :)